travis-scripts/nightly/create-tag.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+if [[ $TRAVIS_EVENT_TYPE != *"cron"* ]]; then
+  exit 0
+fi
+
```

Comment:
We need to add checks:

* Run this script in ONLY one job
* Check `GITHUB_OAUTH_TOKEN` exists